tests/manage/monitoring/prometheus/test_deployment_status.py

```diff
@@ -122,6 +126,7 @@ def test_ceph_mons_quorum_lost(measure_stop_ceph_mon, threading_lock):
 @tier4c
 @pytest.mark.polarion_id("OCS-900")
 @skipif_managed_service
+@runs_on_provider
```

Comment:
Is this marker correct?

Reply:
earlier marker reference : https://github.com/red-hat-storage/ocs-ci/blob/e73ff0c0638ae1e1e0acc2aee18888c5e59717aa/tests/manage/pv_services/test_del_mon_service_and_create_pvc.py#L55